### PR TITLE
O3-1679: Test results header in the "Test Results" dashboard appearing twice

### PR DESCRIPTION
**Pull request by njiddasalifu**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
 For changes to apps
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary
Patient test results title was duplicated as Test results and Results. The required title is Results only.

## Screenshots

Before

before

After

after

## Related Issue
https://issues.openmrs.org/browse/O3-1679

### CODE REVIEW

**Comment by njiddasalifu** [comment]
Hi @vasharma05 can you please review this PR

**Review by vasharma05 — APPROVED** [review]
Looks good to me!
Thanks!

File: packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.component.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ export interface DashboardConfig {
   slot: string;
   title: string;
   columns: number;
+  hideDashboardTitle: boolean;
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.component.tsx:22]: `hideDashboardTitle` should be an optional property.

**Comment by denniskigen** [comment]
Thanks, @njiddasalifu!

**Pull request merged**